requirements.txt

```diff
@@ -2,3 +2,4 @@ ocpp
 cryptography
 pydantic==1.*
 websockets
+py4j
```

Comment:
Why is py4j added here?

Reply:
The py4j module is used to execute the ExiCodec.jar from Josev. I must have forgotten this module :sweat_smile: